src/users/__tests__/usersReducer-test.js

```diff
@@ -99,13 +99,18 @@ describe('usersReducer', () => {
       check({ timezone: randString() });
     });
 
-    // Excluded: "When the owner of a bot changes." The `users` state
-    // doesn't include cross-realm bots.
+    test.skip('When the owner of a bot changes.', () => {
+      check({ user_id: theUser.user_id, bot_owner_id: (theUser.bot_owner_id ?? 1) + 1 });
```

Comment:
nit: this and the rest of this commit still introduce setting user_id, then that's cut in a later commit